src/components/LoadingMessage/LoadingMessage.spec.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import assert from 'assert';
+import { common } from '../../util/generic-tests';
+import LoadingMessage from './LoadingMessage';
+import LoadingIcon from '../Icon/LoadingIcon/LoadingIcon';
+import SuccessIcon from '../Icon/SuccessIcon/SuccessIcon';
+
+const {
+	Icon,
+	Title,
+	Body,
+} = LoadingMessage;
+
+describe('LoadingMessage', () => {
```

Comment:
this isn't a big deal, but this test breaks the pattern of tests we have for most other components:

```
describe('component')
    describe('render')
    describe('props')
    describe('child components')
```